docs/modules/getting-started/pages/security.adoc

```diff
@@ -0,0 +1,337 @@
+= Authenticate Client Connections
+:description: This tutorial guides you on securing the communication between a cluster member and client using the security feature's identity configuration.
+
+{description} This way the member will accept and authorize connections from the clients which have the matching identity configurations.
+We recommend you to secure your deployments; without security, unauthorized members or clients may access
+the cluster and use the cluster resources.
+
+In this tutorial, you will:
+
+. Start a Hazelcast Enterprise member with security enabled and configured
+. Connect a client which is configured to be authorized
+. Try to connect another client which does not respect to the security requirements
+
+== Before You Begin
+
+To complete this tutorial, you need the following:
+
+[cols="1a,1a"]
+|===
+|Prerequisites|Useful resources
+
+|Trial license
+|https://trialrequest.hazelcast.com/[Hazelcast website]
+
+|Docker image for the full Hazelcast distribution
+|xref:get-started-enterprise.adoc[Install Hazelcast Enterprise]
+
+|===
+
+== Step 1. Configure the Enterprise Member
+
+Since the configuration for security is more detailed, you'll use a Hazelcast configuration file
+and pass it to the Docker command, instead of giving all the configuration options in a single Docker command.
+
+In an existing or new directory, place a `hazelcast.yaml` (or `xml`) file with the following content.
```

Comment:
```suggestion
In an existing or new directory, create a `hazelcast.yaml` (or `xml`) file, and add the following:
```

Reply:
The Docker `run` command mounts the `~/config` directory. Shouldn't we tell users to put their config file there? A tutorial should be opinionated to allow users to easily copy-paste and not make decisions.